addons/ai/functions/fnc_unpackStaticWeaponPFH.sqf

```diff
@@ -0,0 +1,68 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Move units together if needed and unpacks a static weapon from units' backpacks.
+ *
+ * Arguments:
+ * 0: Gunner <OBJECT>
+ * 1: Assistant <OBJECT>
+ * 2: Target Position <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_gunner, _assistant] call zen_ai_fnc_unpackStaticWeaponPFH
+ *
+ * Public: No
+ */
+
+params ["_gunner", "_assistant", ["_targetPos", [], [[]], 3]];
+
+if !(local _gunner) exitWith {
+    [QGVAR(unpackStaticWeapon), _this, _gunner] call CBA_fnc_targetEvent;
+};
+
+_gunner setVariable [QGVAR(unpackStaticWeaponTargetPos), _targetPos];
+
+// Close enough, set up weapon
+if (_gunner distance _assistant < 3) exitWith {
+    [_gunner, _assistant] call FUNC(unpackStaticWeapon);
+};
+
+// Too far, order assistant to move to gunner and start pfh
+_gunner disableAI "PATH";
+[_assistant] joinSilent grpNull;
+private _g = (group _assistant);
```

Comment:
Use the full word `_group` as the variable name. Also, no parentheses around unary commands.